From: Christopher Hoskin <mans0954@debian.org>
Date: Wed, 9 May 2018 20:33:52 +0100
Subject: Revert "Merge pull request #123 from mono/use_msbuild"

This reverts commit e704109318cc8461324684d76dfabe19c5e57ba9, reversing
changes made to 122456e641dca7099748d9214a0c409e61263eb6.
---
 build/xbuild.include | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/xbuild.include b/build/xbuild.include
index 8b9f3d0..8faf48b 100644
--- a/build/xbuild.include
+++ b/build/xbuild.include
@@ -1,4 +1,4 @@
-XBUILD=msbuild
+XBUILD=xbuild
 XBUILD_ARGS=/verbosity:normal /nologo /property:CodePage=65001
 ALL_CSPROJ=$(wildcard *.csproj)
 MAIN_SLN=$(top_builddir)/F-Spot.sln
